assertk-common/src/main/kotlin/assertk/assert.kt

```diff
@@ -164,6 +165,20 @@ fun <T> assert(actual: T, name: String? = null): Assert<T> = assertThat(actual,
  */
 fun <T> assertThat(actual: T, name: String? = null): Assert<T> = ValueAssert(actual, name, null)
 
+/**
+ * Asserts on the given property value with an optional name.
```

Comment:
Should better explain why you'd use this version, particularly that it uses the property's name by default.

Reply:
Changed the comment.